scipy/ndimage/_interpolation.py

```diff
@@ -877,7 +877,7 @@ def rotate(input, angle, axes=(1, 0), reshape=True, output=None, order=3,
     >>> ax2.set_axis_off()
     >>> ax3.imshow(full_img_45, cmap='gray')
     >>> ax3.set_axis_off()
-    >>> fig.set_tight_layout(True)
+    >>> fig.set_layout_engine('tight')
```

Comment:
Why not just `tight_layout`? My understanding of the other method is that it's to use when you redraw and want to again adjust the layout, which is not needed here.

Reply:
I just used the recommendation from the deprecation message, and it seems to be the equivalent thing. I'm not really interested in optimizing this, but if you care then please feel free to push to my branch.

Reply:
I don't particularly care. I just thought you could have done that intentionally instead of the other form and wanted to check :)